src/components/Router/index.js

```diff
@@ -291,6 +303,38 @@ class HistoryRouter extends Component {
     this.props.options.events.emit('complete', data)
   }
 
+  triggerOnError = (apiResponse) => {
+    const { status, response } = apiResponse
+    const error = response.error || {}
+    let message;
+    let type;
+
+    if (error || response.reason) {
+      // The API usually returns a payload with an `error`
+      // key that includes a `message` and `type`
+      // For an authorization error the response would look like this
+      // `status: 401,
+      //  response: {
+      //   error: {
+      //     message: "Authorization error: please re-check your credentials"
+      //     type: "authorization_error"
+      //   }
+      // }`
+      // When requesting video challenges, the response looks like this
+      // `status: 403,
+      //  response: {
+      //   reason: "invalid_token",
+      //   status: "error"
+      // }`
+      // Another important thing to notice is the status code is different!
```

Comment:
Would've been nice if the error response structure from back end was consistent 😞

Reply:
@joninvski is kindly making the response consistent for us ❤️